src/styles/blocs/header.scss

```diff
@@ -0,0 +1,66 @@
+.header {
+  display: flex;
+  height: $header-height;
+  background-color: #fff;
+
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+}
+
+.nav {
+  display: flex;
+  align-items: center;
+  width: 100%;
+  height: 100%;
+  margin: auto 50px;
+  justify-content: space-between;
+
+  &__logo {
+    display: flex;
+  }
+
+  &__list {
+    display: flex;
+    height: 100%;
+    margin: 0;
+    padding-inline-start: 0;
+  }
+
+  &__list > :not(:last-child) {
+    margin-right: 20px;
+  }
+
+  &__item {
+    display: flex;
+    list-style: none;
+  }
+
+  &__link {
+    color: $main-text-color;
+    text-decoration: none;
+    display: flex;
+    align-items: center;
+
+    transition: 0.3s;
+
+    &:hover {
+      color: $hovering-color;
+      transition: 0.3s;
+    }
+  }
+}
+
+.is-active {
+  color: $hovering-color;
+  position: relative;
+
+  &::after {
+    content: "";
+    height: 4px;
+    border-radius: 8px;
```

Comment:
Let's try to organize your CSS
I recommend you take a look at these styles code and use it in the feature
[9elements](https://9elements.com/css-rule-order) or [htmlAcademy](https://codeguide.academy/html-css.html#css-order)